workflow/installertest.groovy

```diff
@@ -0,0 +1,133 @@
+// Replace colons in image with hyphens and append text
+String convertImageNameToString(String imageName, String append="") {
+    return (imageName+append).replaceAll(':','-')
+}
+
+/**
+ * Extracts the components from an {@code 'artifact://full/path/to/job/buildNr#artifact.ext'} type url.
+ * @param url the url
+ */
+@NonCPS
+def getComponentsFromArtifactUrl(String url) {
```

Comment:
This is freakin' handy, and we should really be moving it into `Workflow` proper, IMO, in some way. Well, ok, actually `fetch_artifact` too. Ok, moving on, ignore me for now. =)